core/flyout_horizontal.ts

```diff
@@ -267,29 +267,35 @@ export class HorizontalFlyout extends Flyout {
     for (let i = 0, item; (item = contents[i]); i++) {
       if (item.type === 'block') {
         const block = item.block;
-        const allBlocks = block!.getDescendants(false);
+        // CHANGE: Checked whether block is undefined:
+        if (typeof block === "undefined") {
+          continue;
+        }
+        // CHANGE: Removed ! from after the remaining uses of block within the for loop.
```

Comment:
You don't need to describe code changes as comments in the code. Just describe what the code does as you've written it. Any such comments will continue to exist after your code is merged and should describe the code as it currently exists, not how it differs from previous versions.